contracts/treasury/TreasuryImplementation.sol

```diff
@@ -247,27 +295,16 @@ contract TreasuryImplementation is
 
     function getConvertedAmount(address _tokenAddress, uint256 _amount)
         external
-        view
```

Comment:
because this calls a method from IQuoter that is not a market as 'view':
/// @title Quoter Interface
/// @notice Supports quoting the calculated amounts from exact input or exact output swaps
/// @dev These functions are not marked view because they rely on calling non-view functions and reverting
/// to compute the result. They are also not gas efficient and should not be called on-chain.
interface IQuoter {

It's a mistake from uniswap. 
The method is a view but it cannot be market as a view.